airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/common.py

```diff
@@ -35,9 +43,10 @@ async def _run(self, *args: Any) -> StepResult:
         build_results_per_platform = {}
         for platform in self.build_platforms:
             try:
-                connector = await self._build_connector(platform, *args)
+                connector_container = await self._build_connector(platform, *args)
+                connector_container = apply_airbyte_docker_labels(connector_container, self.context.connector)
```

Comment:
@alafanechere I had an idea to move the application of the docker labels to a generic place for all connectors.

WDYT?